### ISSUE TITLE
Add possibility to remove ASG and Launch configuration from the API

### ISSUE THREAD

**Issue by mvisonneau** [trainline/environment-manager#123]
Whenever we want to completely redeploy a stack, it would be great if we could do it from the API and the UI instead of having to do it manually from the AWS CLI or console

**Comment by duncanhall** [comment]
I believe this is currently "as designed" because Environment Manager should not just enable things you can already do via the AWS console.

As we move towards  the "Environment refresh" features, functionality to clear entire ASGs will obviously be needed, but it might not necessarily be exposed as a "delete ASG" feature (rather, it might be part of a process to "install all latest versions of all services on this ASG").

**Comment by mvisonneau** [comment]
Yes absolutely, I was seeing this more as "clear service" and "clear stack" functionalities.
